source/android-source/pilgrimunitysdk/src/main/java/com/foursquare/pilgrimunitysdk/PilgrimUnitySDK.java

```diff
@@ -13,7 +14,10 @@ private PilgrimUnitySDK() {
     }
 
     public static void init(@NonNull Context context, @NonNull String consumerKey, @NonNull String consumerSecret) {
-        PilgrimSdk.with(new PilgrimSdk.Builder(context).consumer(consumerKey, consumerSecret));
+        PilgrimSdk.with(new PilgrimSdk.Builder(context)
+                .consumer(consumerKey, consumerSecret)
+                .logLevel(LogLevel.DEBUG)
```

Comment:
Not sure if we want to make this configurable, as it is now it will enable debug logging always, same for iOS: https://github.com/foursquare/pilgrim-unity-sdk/blob/feature/debug-screen/source/pilgrim-unity-sdk/Assets/PilgrimUnitySDK/Plugins/iOS/PilgrimUnitySDK.m#L14

Reply:
i think thats ok. its light weight and the data goes away after some limit (2 days on android, and i think 1k records on ios)